gnomad_qc/v3/resources.py

```diff
@@ -1,131 +1,153 @@
+qc_ac = TableResource('gs://gnomad/annotations/hail-0.2/ht/genomes_v3/gnomad_genomes_qc_ac.ht')
+fam_stats = TableResource('gs://gnomad/annotations/hail-0.2/ht/genomes_v3/gnomad_genomes_qc_fam_stats.ht')
+
+# TODO: All this is now in gnomad_hail --> Adapt variant QC
+# truth_samples={
+#     'syndip': {
+#         's': "CHMI_CHMI3_WGS2",
+#         'truth_mt': syndip_mt_path,
+#         'v3_mt': v3_syndip_mt_path,
+#         'bed': syndip_bed_path
+#     },
+#     'na12878': {
+#         's': "NWD714003",
+#         'truth_mt': na12878_giab.path,
+#         'v3_mt': v3_na12878_mt_path,
+#         'bed': na12878_bed_path
+#     }
+# }
+
+
+def _get_ancestry_pca_ht_path(part: str, include_unreleasable_samples: bool = False) -> str:
     return 'gs://gnomad/sample_qc/ht/genomes_v3/gnomad_v3_pca_{0}{1}.{2}'.format(
         part,
         '_with_unreleasable_samples' if include_unreleasable_samples else '',
         'txt' if part == 'eigenvalues' else 'ht'
     )
 
 
-def get_project_meta() -> hl.Table:
-    return hl.import_table(project_meta_path, impute=True, key='s')
+def get_ancestry_pca_loadings(include_unreleasable_samples: bool = False) -> TableResource:
+    return TableResource(_get_ancestry_pca_ht_path('loadings', include_unreleasable_samples))
 
 
-def get_full_mt_path(split: bool = True) -> str:
-    split_str = '.split' if split else ''
-    return f"gs://gnomad/raw/hail-0.2/mt/genomes_v3/gnomad_genomes_v3.repartitioned{split_str}.mt"
+def get_ancestry_pca_scores(include_unreleasable_samples: bool = False) -> TableResource:
+    return TableResource(_get_ancestry_pca_ht_path('scores', include_unreleasable_samples))
 
 
-def get_full_mt(
-        split: bool = True,
+def get_ancestry_pca_eigenvalues_path(include_unreleasable_samples: bool = False) -> str:
+    return _get_ancestry_pca_ht_path('scores', include_unreleasable_samples)
+
+
+def get_gnomad_v3_mt(
         key_by_locus_and_alleles: bool = False,
         remove_hard_filtered_samples: bool = True,
-        release_only: bool = False
+        release_only: bool = False,
+        samples_meta: bool = False
 ) -> hl.MatrixTable:
-    mt = hl.read_matrix_table(get_full_mt_path(split))
+    mt = gnomad_v3_genotypes.mt()
     if key_by_locus_and_alleles:
         mt = hl.MatrixTable(hl.ir.MatrixKeyRowsBy(mt._mir, ['locus', 'alleles'], is_sorted=True))
 
     if remove_hard_filtered_samples:
-        hard_filtered_ht = hl.read_table(hard_filtered_samples_ht_path)
-        mt = mt.filter_cols(hl.is_missing(hard_filtered_ht[mt.col_key]))
+        mt = mt.filter_cols(hl.is_missing(hard_filtered_samples.ht()[mt.col_key]))
+
+    if samples_meta:
+        mt = mt.annotate_cols(meta=meta.ht()[mt.col_key])
+
+        if release_only:
+            mt = mt.filter_cols(mt.meta.release)
+
+    elif release_only:
+        mt = mt.filter_cols(meta.ht()[mt.col_key].release)
 
     return mt
 
 
-def sample_qc_path(strat: str = ""):
+def get_sample_qc(strat: str = "") -> TableResource:
```

Comment:
what is strat here?

Reply:
Good point, let me add comments on these public functions